Darwin/Services/StitchRemoteMongoDBService/StitchRemoteMongoDBService/Internal/InternalChangeStreamDelegate.swift

```diff
@@ -6,11 +7,11 @@ import StitchCoreRemoteMongoDBService
  * Internal class which serves as an intermediary between an internal RawSSEStream and a change stream as exposed
  * by the remote MongoDB service.
  */
-internal class InternalChangeStreamDelegate<PublicDelegateT: ChangeStreamDelegate>: SSEStreamDelegate {
-    public weak var delegate: PublicDelegateT?
+internal class InternalChangeStreamDelegate<T: Codable>: SSEStreamDelegate {
+    public var delegate: ChangeStreamType<T>?
```

Comment:
[naming nit] should probably not be called `delegate`, this would also probably let you get rid of the swiftlint exception

Reply:
+1 delegate.delegate seems funny